RELEASE_NOTES.md

```diff
@@ -36,33 +38,33 @@ accompanying blog post.
 
 - New Sigmoid display transform module.
 
-- Two new highlight recovery algorithm are proposed: in-paint opposed
+- Two new highlight recovery algorithms are proposed: in-paint opposed
   and segmentation. The in-paint opposed has proved to be very stable
-  and good in a lot of pictures so it has been made the default now.
+  and good in a lot of pictures, so it has been made the default now.
 
 - A full rework of the image display has been done. An initial work
-  has been to create an easy to use image display on the central
-  area. This routine supports all display mode: focus-peaking and
+  has been to create an easy-to-use image display on the central
+  area. This routine supports both focus-peaking and
   color assessment. This routine is now used for the darkroom area,
-  for the second window and the duplicates.
+  for the second window, and the duplicates.
 
-  So the second window gets the focus-peaking display and more
+  So, the second window gets the focus-peaking display and more
   importantly the color assessment mode which is an important feature.
 
-  To achieve that properly a new routine has been created which run a
+  To achieve that properly, a new routine has been created which runs a
   separate pixelpipe which is fully equivalent to the darkroom
   one. This gives a properly aligned display.
 
-  Using the new pixelpipe and display routines the snapshot routine
+  Using the new pixelpipe and display routines, the snapshot routine
   has been reworked. This new version has fully dynamic snapshots. The
   snapshot can be panned and zoomed and still perfectly aligned with
-  the darkroom display which permit a proper comparison.
+  the darkroom display which permits a proper comparison.
 
 - Add style preview in lighttable style module and in darkroom style
-  widget. The preview is displayed in the tooltip when overing the
+  widget. The preview is displayed in the tooltip when hovering over the
```

Comment:
covering, hovering?

Reply:
hovering, for sure. But hovering is sufficient, no need to add over

Reply:
Are you sure?
https://www.merriam-webster.com/dictionary/hover
'hover', as an _intransitive_ verb, means:
- to hang fluttering in the air or on the wing: _A hawk hovered overhead._
- to remain suspended over a place or object: _a hummingbird hovering over the flowers_; _Helicopters hovered above us._
- to move to and fro near a place : fluctuate around a given point: _Unemployment hovered around 10 percent._
- to be in a state of uncertainty, irresolution, or suspense: _hovering uncomfortably behind a cigar_, _The country hovers on the brink of famine._

The _transitive_ verb needs a preposition like _over_, _above_, _between_:
to position (a computer cursor) over something (such as an image or icon) without selecting it: _Many in the class hovered their cursors over words and icons for long periods before committing to clicking their mouse._
See examples at https://context.reverso.net/translation/english-german/hover and https://www.linguee.com/english-german/search?source=auto&query=hover (the corpus database behind DeepL.com).

Reply:
"hovering over" is correct